internals/secrethub/fakes/timeformatter.go

```diff
@@ -1,4 +1,4 @@
-// +build !production
+//go:build !production
```

Comment:
Please confirm that we don't need both of these build tags. We had an issue a while back on the 1P CLI for not including both.

Reply:
Looking at the [tag transition](https://stackoverflow.com/a/68361186), it feels like the safest way is to upgrade to Go 1.18 and run `go fix`. This should address the concern that you have. What do you think?